### PR TITLE
optimize columnHeaderRenderer method

### PR DESCRIPTION
**Pull request by albertywu**
50% faster using native javascript API vs old jQuery way

### CODE REVIEW

**Comment by albertywu** [comment]
@SimplGy @bcherny

File: slick.grid.js [hunk 1]
```diff
@@ -800,10 +800,17 @@ if (typeof Slick === "undefined") {
...
-      return $el;
+
+      // 50% faster using native API, versus the old jQuery way
+      // jQuery's .html() is particularly slow
+      //
+      // Saves roughly 0.5ms * N in synchronous processing time, where N is the number of columns
+      var d = document.createElement('div')
+      d.className = 'cell';
+      d.innerHTML = "<span class='name'>" + column.name + "</span>";
+      d.title = column.toolTip || "";
```
Comment (SimplGy) [inline slick.grid.js]: I don't know if setting the title attr is expensive, but do you want it to be a blank string when falsey? I'd lean towards `if (column.toolTip) { d.title = column.toolTip }

**Comment by SimplGy** [comment]
:+1: this is great, thanks

File: slick.grid.js (hunk 1 above)
Comment (albertywu) [inline slick.grid.js]: good call, will update

**Pull request merged**